MonstieWash/Assets/Scripts/TaskTracker.cs

```diff
@@ -0,0 +1,122 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using System.Linq;
+
+public class TaskTracker : MonoBehaviour
+{
+    //Unity Inspector
+    //Container object for all the the different areas to clean 
+    [SerializeField] private Transform taskContainer;
+    //These are temporary replacement for UI progress bar
+    [SerializeField] private SerializableDictionary<string, float> m_taskProgress = new SerializableDictionary<string, float>();
+
+    //Private
+    private List<string> m_taskDictKeys = new List<string>();
```

Comment:
Same with here, try to use new().
```suggestion
    private List<string> m_taskDictKeys = new();
```